composer.json

```diff
@@ -23,7 +23,7 @@
     ],
     "require": {
         "php": ">=8.1",
-        "psr/log": "^1.0 || ^2.0",
+        "psr/log": "^1.0 || ^2.0　|| ^3.0",
```

Comment:
Should be `"^1.0 || ^2.0 || ^3.0"`.
There is a zero width space (`　`) after `2.0`.